test.js

```diff
@@ -48,4 +48,18 @@ describe('csslint-stylish', () => {
 
     assert(CSSLint.hasFormat('stylish'), 'csslint should be stylish')
   })
+
+  it('should not report undefined output lines when no filename provided', () => {
```

Comment:
Line is too long.<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).